### ISSUE TITLE
Using the same key for different controls

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2336]
We are using the same text key for the Close button in top left of load window, and the Collapse button on the Add resource form. 

<img width="135" alt="Screen Shot 2019-03-28 at 11 36 28 AM" src="https://user-images.githubusercontent.com/6945691/55183821-e2bed680-514d-11e9-87ae-6bc7846d329b.png">
<img width="123" alt="Screen Shot 2019-03-28 at 11 36 35 AM" src="https://user-images.githubusercontent.com/6945691/55183814-e0f51300-514d-11e9-9dfd-a640cbf6ef32.png">  

These don't do the same thing and should be able to have separate tooltips/aria labels. We have a lot of "close" actions in the app and we're trying to be specific about what will be affected by each one. Can these be separated, please?

Load window close - "Close resource list"
Add resource form - "Close form"
